src/App/App.js

```diff
@@ -14,12 +14,26 @@ function App() {
     setPosters(moviePosters);
   }, [] );
 
+  function changeVoteCountData(id, vote) {
+    setPosters((prevMoviePosters) => {
+      return prevMoviePosters.map((movie) => {
+        if (movie.id === id) {
+          return {
+            ...movie,
+            vote_count: vote === "up" ? movie.vote_count + 1 : movie.vote_count - 1
+          };
+        }
+        return movie;
+      });
+    });
+  }
+
```

Comment:
Fantastic job getting the voting functionality up and running! This looks very clean.